src/core/lib/surface/call.cc

```diff
@@ -1892,13 +1913,15 @@ static grpc_call_error call_start_batch(grpc_call* call, const grpc_op* ops,
   if (!is_notify_tag_closure) {
     GPR_ASSERT(grpc_cq_begin_op(call->cq, notify_tag));
   }
-  gpr_ref_init(&bctl->steps_to_complete, num_completion_callbacks_needed);
+  gpr_ref_init(&bctl->steps_to_complete, has_send_ops + num_recv_ops);
```

Comment:
Can we do something more explicit, rather than adding bools and ints?

Reply:
What approach would you consider more explicit?  Would it address your concern to simply do an explicit cast of the bool to an int?  Or maybe something like `has_send_ops ? 1 : 0`?

Reply:
Cast is fine, I think the ternary operator is a little more explicit though.

Reply:
Done.